package/samplers/hebo/sampler.py

```diff
@@ -1,12 +1,18 @@
 from __future__ import annotations
 
+from typing import Optional
```

Comment:
We should not use Optional. We should use `None | T` instead of `Optional[T]`.

Reply:
Fixed.